File: UKParliament.CodeTest.Web/ViewModels/PersonViewModel.cs
using System.ComponentModel.DataAnnotations;

namespace UKParliament.CodeTest.Web.ViewModels;

public class PersonViewModel
{
    public int Id { get; set; }

    public string Title { get; set; }

    public string FirstName { get; set; }

    public string LastName { get; set; }

    public DateTime DateOfBirth { get; set; }

    public string Gender { get; set; }
}

File: UKParliament.CodeTest.Web/ClientApp/src/app/models/person-view-model.ts
//export interface PersonViewModel {
//  id: number;
//}

export class Person {
  id?: number;
  title?: string;
  firstName?: string;
  lastName?: string;
  dateOfBirth?: Date;
  gender?: any;
}

export enum Gender {
  Male = 1,
  Female = 2
}
